.github/workflows/ar-pr-deployment.yml

```diff
@@ -17,20 +17,12 @@ jobs:
         steps:
             - uses: actions/checkout@v2
             - uses: guardian/actions-setup-node@main
-
-            - uses: actions/cache@v2
-              with:
-                  path: ~/.npm
-                  key: node-${{ hashFiles('**/package-lock.json') }}
-                  restore-keys: node-
             - name: install
-              run: npm ci
-              working-directory: apps-rendering
+              run: yarn --frozen-lockfile
             - name: run
-              working-directory: apps-rendering
               run: |
                   npm install -g ngrok
```

Comment:
worth converting this to `yarn` while we're at it?

Reply:
👍